src/components/flow/actions/sendbroadcast/SendBroadcastForm.tsx

```diff
@@ -71,7 +64,7 @@ export default class SendBroadcastForm extends React.Component<
     if (this.state.template.value !== null) {
       fetchAsset(this.props.assetStore.templates, this.state.template.value.uuid).then(
         (asset: Asset) => {
-          if (asset !== null) {
+          if (asset && asset.content) {
```

Comment:
I think you can also use like this asset?.content

Reply:
@pankaj-ag optional chaining is released in typescript version 3.7. Here we are using 3.4.

Will update this when floweditor updates the typescript version.